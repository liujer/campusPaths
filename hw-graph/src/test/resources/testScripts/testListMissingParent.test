# Tests listing of children of a missing parent node
CreateGraph C
ListChildren C one