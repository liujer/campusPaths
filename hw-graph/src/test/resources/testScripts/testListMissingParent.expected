# Tests listing of children of a missing parent node
Created graph C
one does not exist in graph C